tests/components/webostv/test_config_flow.py

```diff
@@ -44,25 +43,6 @@ async def test_form(hass: HomeAssistant, client) -> None:
     """Test we get the form."""
     assert client
```

Comment:
why the assert btw?

Reply:
Not needed, removed, this code as gone so many changes only God and I knows why, and I forgot why 🐱